src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -187,16 +187,17 @@
   <data name="DestructiveChangesNotification" xml:space="preserve">
     <value>Creating this template will make changes to existing files:</value>
   </data>
-  <data name="DisplaysHelp" xml:space="preserve">
```

Comment:
We previously agreed not to remove yet the localized strings that are already there. 
Should we reconsider the decision with our new knowledge about localization?

Reply:
I can keep old names, however the names are very different for each option:
- DisplaysHelp is used for --help option description
- ShowFilteredTemplates is used for --type option description 
and so on.
It's much convenient to have them unified.